partition/bootloader.go

```diff
@@ -43,27 +42,18 @@ var (
 	ErrBootloader = errors.New("cannot determine bootloader")
 )
 
-type bootloaderName string
-
 type bootLoader interface {
```

Comment:
Can we rename this to bootloaderIface or similar?  The subtle upper casing there is a bit cryptic and unfriendly to usual conventions.